controller/pods.go

```diff
@@ -251,6 +291,7 @@ func (c *Controller) watchPods(ctx context.Context, nodeName string) (cache.Cont
 		&apiv1.Pod{},
 		resyncPeriod,
```

Comment:
According to the comment above, this one may explain why the pod creation contains the new network annotations.